### PR TITLE
More wheels: manylinux and Mac OS

### PR DESCRIPTION
**Pull request by mgedmin**
I've also switched pypy images from old pypy3.5-6.0 to modern pypy3, and fixed a regression there (dict view iteration failed with a ForbiddenAttribute).

### CODE REVIEW

**Comment by mgedmin** [comment]
I'm testing a workaround for the Mac OS Python 3.5 problem.

The PyPy3 problem looks new, probably caused by a newer PyPy3 than we tested before.

File: src/zope/security/checker.py [hunk 1]
```diff
@@ -886,6 +886,10 @@ def _fixup_dictlike(dict_type):
             o_type = type(obj)
             if o_type not in _default_checkers:
                 _default_checkers[o_type] = _iteratorChecker
+            # PyPy3 has special types for iter({}.items()) etc.
+            iter_type = type(iter(obj))
+            if iter_type not in _default_checkers:
+                _default_checkers[iter_type] = _iteratorChecker
```
Comment (mgedmin) [inline src/zope/security/checker.py]: Note to self: this needs a changelog note.

**Comment by mgedmin** [comment]
PyPy3 should be fixed now.

**Comment by mgedmin** [comment]
Coveralls is showing bad numbers and should feel bad.

File: .travis.yml [hunk 2]
```diff
@@ -1,31 +1,108 @@
...
+  - python setup.py bdist_wheel
+
 after_success:
-    - coveralls
+  - coveralls
+  - |
+    if [[ $TRAVIS_TAG && "$TRAVIS_OS_NAME" == "osx" ]]; then
+      # macpython 3.5 doesn't support recent TLS protocols which causes twine
+      # upload to fail, so we use the system Python to run twine
+      /usr/bin/python -m pip install twine
```
Comment (mgedmin) [inline .travis.yml]: This needs a `/usr/bin/python -m ensurepip` first.

Comment (mgedmin) [reply .travis.yml]: Actually it needs much more, see https://github.com/zopefoundation/zope.interface/pull/148.  (I'd prefer to release zope.interface 1.7.1 to make sure that recipe works before copying it elsewhere.)

**Comment by mgedmin** [comment]
Rebased on master, fixed Mac OS twine installation.

File: .travis.yml [hunk 3]
```diff
@@ -1,31 +1,116 @@
...
-    - python: 3.7
+
+    - name: "Python: 3.7, pure (no C extensions)"
+      python: 3.7
       env: PURE_PYTHON=1
+
+    # manylinux wheel builds
+    - name: 64-bit manylinux wheels (all Pythons)
+      services: docker
+      env: DOCKER_IMAGE=quay.io/pypa/manylinux2010_x86_64
```
Comment (jamadden) [inline .travis.yml]: 👍

**Pull request merged**